kolibri/plugins/facility/assets/src/views/users/NewUsersPage.vue

```diff
@@ -218,6 +236,39 @@
         noNewUsersDescription$,
       };
     },
+    computed: {
+      hasSelectedUsers() {
+        return this.selectedUsers && this.selectedUsers.size > 0;
+      },
+      listContainsLoggedInUser() {
+        return this.selectedUsers.has(this.currentUserId);
+      },
+      canAssignCoaches() {
+        if (!this.hasSelectedUsers) return false;
+        return this.facilityUsers
```

Comment:
@nucleogenesis Just a note that we will need to solve this in a more robust way, since this array of `facilityUsers` only contains the users of the current page, so we can skip the validations if a selected user is in another page. Also, it'd be great if we can share these conditions for both pages, the new users and the users root:

https://github.com/user-attachments/assets/2966f777-d6fb-4f94-bc2a-94af6ccfde7f